public/stylesheet.css

```diff
@@ -1,11 +1,14 @@
 * {
   box-sizing: border-box;
+  font: inherit;
   margin: 0;
+  outline: 0;
```

Comment:
I understand your motivation to move `outline: 0` here, but in general it's not a good practice to remove `outline` from all (or any) elements - [read more about it on this website](http://www.outlinenone.com/). Could you revert the changes related to the `outline` property, and possibly setting them all to `none` instead of `0` ([ref](https://stackoverflow.com/questions/35648667/outline-none-vs-outline-0)).

The motivation for having this rule on some elements is that they have a clear focus state (though this is topic that's for debate if any a11y experts disagre).

Reply:
thanks for that, I would make this changes.